### PR TITLE
Add documentation for the noboot option

### PR DESCRIPTION
**Pull request by whowutwut**
In reference to the changes for #1881 

Add documentation describing the noboot attribute for Minsky Servers
In the process, cleaned up some of the text for the site table man page

### CODE REVIEW

**Comment by whowutwut** [comment]
@zet809  please take a look and let me know if i should add some more information in some other location in the documentation.  I'm also not sold on the wording of the noboot section...

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/configure/networks.rst [hunk 1]
```diff
@@ -33,12 +37,22 @@ Initialize DHCP services
 
 #. Configure DHCP to listen on different network interfaces (**Optional**)
 
-   xCAT allows specifying different network intercaces thateDHCP can listen on for different nodes or node groups.  If this is not needed, go to the next step.  To set dhcpinterfaces :: 
+   The ``dhcpinterfaces`` keyword allows users to specify or limit the DHCP to listen over certain network interfaces.
```
Comment (zet809) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/configure/networks.rst]: Do we need to mention if no 'dhcpinterfaces' specified, what shall be the default value? The nics in networks table.

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/configure/networks.rst [hunk 2]
```diff
@@ -33,12 +37,22 @@ Initialize DHCP services
...
+   If the management node has 4 interfaces, (eth0, eth1, eth2, and eth3), and you want DHCP to listen only on "eth1" and "eth3", set ``dhcpinterfaces`` with: ::
 
-   For more information, see ``dhcpinterfaces`` keyword in the :doc:`site </guides/admin-guides/references/man5/site.5>` table.
+      chdef -t site dhcpinterfaces="eth1,eth3"
+
+   To set "eth1" and "eth3" on the management node, and "bond0" on all the nodes in the "service" nodegroup, set ``dhcpinterfaces`` with: ::
 
+      chdef -t site dhcpinterfaces="xcatmn|eth1,eth3;service|bond0"
+
+   [**noboot**]: For the IBM OpenPower S822LC for HPC ("Minsky") nodes, the BMC and "eth0" on the compute side shares the same physical ethernet port.  However, it is recommended to allow the BMC to be dedicated and to use "eth1" for the compute node.   When an open range is configured on the two networks, the xCAT Genesis Kernel will be sent to the BMC interface and will cause problems with discovery.  In this scenario, if "eth1" is the BMC network and "eth3" is the compute network, disabled genesis by setting ``:noboot`` in ``dhcpinterfaces`` with: ::
```
Comment (zet809) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/configure/networks.rst]: OpenPOWER

Comment (zet809) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/configure/networks.rst]: I'd like to change "However, it is recommended to allow the BMC to be dedicated and to use "eth1" for the compute node." to be "However, it is recommended that BMC port to be dedicated and using 'eth1' for the compute node."
And "In this scenario, if "eth1" is the BMC network and "eth3" is the compute network, disabled genesis by setting" to be "In this scenario, if 'eth1 connect to BMC network and "eth3" connect to the compute network, disabled genesis boot for BMC network by setting"

**Comment by whowutwut** [comment]
@zet809  Thank you!  I made some changes to the section and also put ``noboot`` into a 3rd tier section header so that we can easily point others and link this from our release notes.

**Comment by zet809** [comment]
@whowutwut Thx, victor, looks good to me, I agree to merge it.

**Pull request merged**